x-pack/plugins/monitoring/server/telemetry_collection/get_es_stats.ts

```diff
@@ -93,6 +105,14 @@ export async function fetchElasticsearchStats(
 }
 
 export interface ESClusterStats {
+  elasticsearch?: {
+    cluster: {
+      id: string;
+      stats: {
+        stack: object;
+      };
+    };
+  };
```

Comment:
Makes sense! Happy to keep `stack_stats` independent to avoid breaking changes in the data model.

Regarding `cluster_uuid`, please refer to my comment in `x-pack/plugins/monitoring/server/telemetry_collection/get_all_stats.ts`.